doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -93,7 +93,7 @@ The Fast Pair service implementation provides API to generate the advertising da
   To use this extension, ensure the following:
 
   #. Call :c:func:`bt_fast_pair_battery_set` to provide battery information.
-  #. Set :c:member:`bt_fast_pair_adv_config.adv_battery_mode` to either :c:enum:`BT_FAST_PAIR_ADV_BATTERY_MODE_SHOW_UI_IND` or :c:enum:`BT_FAST_PAIR_ADV_BATTERY_MODE_HIDE_UI_IND` to include the battery notification in the generated advertising payload.
+  #. Set :c:member:`bt_fast_pair_adv_config.not_disc.battery_mode` to either :c:enum:`BT_FAST_PAIR_ADV_BATTERY_MODE_SHOW_UI_IND` or :c:enum:`BT_FAST_PAIR_ADV_BATTERY_MODE_HIDE_UI_IND` to include the battery notification in the generated advertising payload.
```

Comment:
```suggestion
  #. Set :c:member:`bt_fast_pair_adv_config.battery_mode` to either :c:enum:`BT_FAST_PAIR_ADV_BATTERY_MODE_SHOW_UI_IND` or :c:enum:`BT_FAST_PAIR_ADV_BATTERY_MODE_HIDE_UI_IND` to include the battery notification in the generated advertising payload.
```
I think this links correctly to the API docs.

Reply:
I doesn't link either.

Reply:
Tried `bt_fast_pair_adv_config.bt_fast_pair_not_disc_adv_info.battery_mode`. Let's see if it works.

Reply:
`bt_fast_pair_not_disc_adv_info.battery_mode` works. I tried this in my local build and successfully links.

Reply:
Changed to: `Set :c:member:`bt_fast_pair_not_disc_adv_info.battery_mode` in :c:struct:`bt_fast_pair_adv_config` to either...` to be more precise. Do you approve?